spec/openapi.yaml

```diff
@@ -2741,6 +2741,52 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  '/context-api/v2/projects/{projectId}/contexts/remove/async':
+    post:
+      summary: Delete the contexts
+      description: |-
+        Submits async operation to delete the contexts metadata and the content. 
```

Comment:
"Initiates an asynchronous process to delete the contexts."

Reply:
Please update "Delete the context" endpoint and change the first sentence in the description:
> Deletes the context.